src/Controller/OcrController.php

```diff
@@ -220,16 +221,22 @@ public function homeAction(): Response
      */
     public function apiAction(): JsonResponse
     {
-        $this->setup();
-        $result = $this->getResult(EngineBase::WARN_ON_INVALID_LANGS);
-        $responseParams = array_merge(static::$params, [
-            'text' => $result->getText(),
-        ]);
-        $warnings = $result->getWarnings();
-        if ($warnings) {
-            $responseParams['warnings'] = $warnings;
+        try {
+            $this->setup();
+            $result = $this->getResult(EngineBase::WARN_ON_INVALID_LANGS);
+            $responseParams = array_merge(static::$params, [
+                'text' => $result->getText(),
+            ]);
+            $warnings = $result->getWarnings();
+            if ($warnings) {
+                $responseParams['warnings'] = $warnings;
+            }
+            return $this->getApiResponse($responseParams);
+        } catch (Exception $exception) {
+            return $this->getApiResponse([
+                "message" => $exception->getMessage(),
```

Comment:
Other error states return the error in the `error` key, so this should probably do the same, rather than `message`.